doc/guides/source/plugin_contenthandler.textile

```diff
@@ -60,7 +60,32 @@ h4. Using Configuration
 		},
 		handler: {
 			generic: {
-				transformFormattings: false // this will disable the transformFormattings method in the generic content handler
+				// this will disable the transformFormattings method in the generic content handler
+				transformFormattings: false,
+				// this elements (to counter Chrome copy&paste bug) on styled elements
```

Comment:
I know what is meant, but this comment is a bit confusing.

Reply:
I _ a few words :-)